src/string.rs

```diff
@@ -59,6 +62,15 @@ macro_rules! format_eco {
 pub struct EcoString(DynamicVec);
 
 impl EcoString {
+    /// Maximum number of bytes for an inline `EcoString` before spilling on
```

Comment:
General nit-pick: I like to end all doc comments with a dot like a normal sentence.